Basic_viewer/include/CGAL/Drawing_functor.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2022 GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s):   Guillaume Damiand <guillaume.damiand@liris.cnrs.fr>
+//              Mostafa Ashraf <mostaphaashraf1996@gmail.com>
+
+#ifndef CGAL_DRAWING_FUNCTORS_H
+#define CGAL_DRAWING_FUNCTORS_H
+
+#include <CGAL/license/GraphicsView.h>
+#include <CGAL/IO/Color.h>
+#include <functional>
+
+namespace CGAL {
+
+// Drawing functor for a 2D combinatorial data structure
+// (with vertices, edges and faces)
+template <typename DS,
+          typename vertex_handle,
```

Comment:
template parameters should be in CamelCase

Reply:
done